### PR TITLE
Testing navigating property

### PR DESCRIPTION
**Pull request by Luiy0**
### Description

Test navigating property, remove deprecated code, and fixed imports in the `download-preplanned-map` and `display-wfs-layer` samples.

Checklist:
- [ ] Set target branch to main (current release) or v.next (next release)
- [ ] Request a reviewer
- [ ] Assign a reviewer
- [ ] Tag reviewer in comment

### CODE REVIEW

**Comment by Luiy0** [comment]
Assigning @jenmerritt for a first review please.

**Comment by Luiy0** [comment]
Reassigning to @Rachael-E for a first review please.

File: map/download-preplanned-map/src/main/java/com/esri/samples/download_preplanned_map/DownloadPreplannedMapController.java [hunk 1]
```diff
@@ -86,7 +93,7 @@ private void initialize() {
       mapView.getGraphicsOverlays().add(areasOfInterestGraphicsOverlay);
 
       // create a red outline to mark the areas of interest of the preplanned map areas
-      SimpleLineSymbol areaOfInterestLineSymbol = new SimpleLineSymbol(SimpleLineSymbol.Style.SOLID, 0x80FF0000, 5.0f);
+      SimpleLineSymbol areaOfInterestLineSymbol = new SimpleLineSymbol(SimpleLineSymbol.Style.SOLID, Color.web("FF0000", 0.8), 5.0f);
```
Comment (Rachael-E) [inline map/download-preplanned-map/src/main/java/com/esri/samples/download_preplanned_map/DownloadPreplannedMapController.java]: I checked out the docs for `Color.web`, to help keep things human readable, we can use any standard HTML color name here! I've tested it with "red" and that works well here. 

> Creates an RGB color specified with an HTML or CSS attribute string.
> This method supports the following formats:
> Any standard HTML color name

```
      SimpleLineSymbol areaOfInterestLineSymbol = new SimpleLineSymbol(SimpleLineSymbol.Style.SOLID, Color.web("red", 0.8), 5.0f);
```

File: ogc/display-wfs-layer/src/main/java/com/esri/samples/display_wfs_layer/DisplayWFSLayerSample.java [hunk 2]
```diff
@@ -92,10 +93,10 @@ public void start(Stage stage) {
     // make an initial call to load the initial extent's data from the WFS, using the WFS spatial reference
     populateFeaturesFromServer(wfsFeatureTable, initialExtent);
 
-    // use the navigation completed event to populate the table with the features needed for the current extent
-    mapView.addNavigationChangedListener(navigationChangedEvent -> {
+    // use the navigation listener to populate the table with the features needed for the current extent
```
Comment (Rachael-E) [inline ogc/display-wfs-layer/src/main/java/com/esri/samples/display_wfs_layer/DisplayWFSLayerSample.java]:
```suggestion
    // use the navigation property listener to populate the table with the features needed for the current extent
```

**Review by Rachael-E — APPROVED** [review]
thanks @Luiy0 ! Just a couple of suggestions, including Color.web usage and a comment clarification. Happy to this to be merged once that's been fixed up! 

As we discussed, the use of the navigation property listener (and the former listener) in the Download Preplanned Map sample was causing a bug, so my commit has gone ahead and removed that listener and the sample is behaving as expected.

File: map/download-preplanned-map/src/main/java/com/esri/samples/download_preplanned_map/DownloadPreplannedMapController.java (hunk 1 above)
Comment (Luiy0) [reply map/download-preplanned-map/src/main/java/com/esri/samples/download_preplanned_map/DownloadPreplannedMapController.java]: Thanks for the tip, I will adopt it from now on.

**Comment by Luiy0** [comment]
Hi @jenmerritt, this one's ready for a second review please.

**Review by jenmerritt — APPROVED** [review]
Looks fine @Luiy0

**Pull request merged**